build-windows.ps1

```diff
@@ -0,0 +1,34 @@
+$ErrorActionPreference = "Stop"
+
+$version = $args[0]
+
+git checkout "tags/v$version"
+
+$git_success = $?
+
+if (-not $git_success) {
+  throw "Invalid version specified. Specify version as first argument to script"
+}
+
+$env:JAVA_OPTS = "-Xmx4G"
+
+# FIXME should be running tests, not doing so right now because of line ending issues
+sbt clean cliJS/package
+
+$sbt_success = $?
+
+if (-not $sbt_success) {
+  throw "sbt failed"
+}
+
+mkdir target\win
+
+nexe -o target\win\rp.exe cli\js\target\rp.js
+
+$nexe_success = $?
+
+if (-not $nexe_success) {
+  throw "nexe failed"
+}
+
+Compress-Archive -Path .\target\win\rp.exe -DestinationPath ".\target\win\reactive-cli-$version-Windows.zip"
```

Comment:
Is there a naming convention for Windows zip files? I know we're following one for the macOS/Homebrew stuff. Maybe this should be `reactive-cli-<version>-windows-amd64.zip`?

Reply:
Also, wouldn't be surprised if I'm doing something silly in this file, being the first PowerShell script I've written.

Reply:
Your PowerShell looks fine to me. And no, there are no conventions for naming archives.

I was running this on Windows 7, which comes with PowerShell 2.0 and it doesn't have Compress-Archive builtin so the script failed. I think it's okay though, we can make releases on a newer OS. Most importantly, rp.exe runs fine on the same Win7.